<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historial</class>
 <widget class="QDialog" name="historial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>805</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1329</width>
    <height>805</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1329</width>
    <height>805</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tareas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes/imagen.qrc">
    <normaloff>:/p/em.png</normaloff>:/p/em.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="log">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagenes/imagen.qrc">:/p/FOsp.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bus">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>190</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    background-color: #7BDDDD;
    color: black;
    border-radius: 10px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #0EC2C2;
}</string>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="nusu">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; Nombre de usuario&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fonhis">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1411</width>
     <height>931</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Bright</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagenes/imagen.qrc">:/p/gra.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="fdesde">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>d/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tdesde">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Desde&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="thasta">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>200</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Hasta&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>991</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;- Buscar tareas teniendo en cuenta el usuario y fecha de registro&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>1330</width>
     <height>651</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1330</width>
     <height>651</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1330</width>
     <height>651</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>15</number>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>0</height>
    </size>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>132</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>87</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>66</number>
   </attribute>
   <column>
    <property name="text">
     <string>Usuario</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>VLAN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo Tarea</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>No. Tarea</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Equipos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Solicitud</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Soluci√≥n</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Realizada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="cusu">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="registro2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagenes/imagen.qrc">:/p/histo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="fhasta">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>d/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>fonhis</zorder>
  <zorder>log</zorder>
  <zorder>bus</zorder>
  <zorder>nusu</zorder>
  <zorder>fdesde</zorder>
  <zorder>tdesde</zorder>
  <zorder>thasta</zorder>
  <zorder>label</zorder>
  <zorder>tablat</zorder>
  <zorder>cusu</zorder>
  <zorder>registro2</zorder>
  <zorder>fhasta</zorder>
 </widget>
 <resources>
  <include location="imagenes/imagen.qrc"/>
  <include location="imagenes/imagen.qrc"/>
 </resources>
 <connections/>
</ui>
